bookwyrm/importers/calibre_import.py

```diff
@@ -0,0 +1,12 @@
+""" handle reading a csv from calibre """
+from . import Importer
+
+
+class CalibreImporter(Importer):
+    """csv downloads from Calibre"""
+
+    service = "Calibre"
+
+    def get_shelf(self, normalized_row):
+        # Calibre export does not indicate which shelf to use. Go with a default one for now
+        return "to-read"
```

Comment:
Could be `models.Shelf.TO_READ` here (and similar in other importers).

Reply:
that would be better!

Reply:
Updated. Added a test (and updated the importer tests) while I was at it.